operator/pkg/apis/core/v1beta1/configconnectorcontext_types.go

```diff
@@ -42,6 +42,11 @@ type ConfigConnectorContextSpec struct {
 	// Specifies the project to use for preconditions, quota and billing.
 	// Should only be used when requestProjectPolicy is set to BILLING_PROJECT.
 	BillingProject string `json:"billingProject,omitempty"`
+
+	// todo acpana docstring
+	//+kubebuilder:validation:Enum=reconciling;paused
+	//+kubebuilder:validation:Optional
+	ActuationAction string `json:"actuationAction,omitempty"`
```

Comment:
I think we would normally suggest defining an "enum type" here (as much as  go allows)

KRM suggests upper-case constants also.

For example: https://github.com/kubernetes/kubernetes/blob/cb989b84c82ab8c695d02c9a1866bc27de37caec/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go#L252-L258

Technically you can have `ActuationAction ActuationAction json:"actuationAction,omitempty"`, though we normally do something like `Actuation ActuationMode json:"actutation,omitempty"` just to avoid overloading our own mental lexers/parsers